src/runtime/lpython_builtin.py

```diff
@@ -915,6 +915,45 @@ def _lpython_str_split(x: str, sep:str) -> list[str]:
             start += ind + len(sep)
     return res
 
+@overload
+def _lpython_str_replace(x: str, old:str, new:str) -> str:
+    if (old == ""):
+        res: str = ""
+        i: str
+        for i in x:
+            res += new + i
+        return res
```

Comment:
This seems this is already handled in `_lpython_str_replace(x: str, old:str, new:str, count: i32)`, thus seems redundant.

Reply:
Removed.